### PR TITLE
Make Prepare Your Site post shorter by replacing sections to links

### PR DESCRIPTION
**Pull request by BernatBC**
Education section was missing from "Prepare Site" post, so I added it. I also translated it to Spanish.

### CODE REVIEW

**Comment by hossainemruz** [comment]
Thank you @BernatBC. I would like to keep the prepare site doc as small as possible. We already have a dedicated guide for the sections under `Configurations` docs. We should remove all other sections from **Prepare Site** and keep only one (probably **About** section. Then, we can just add list linking to the respective section configuration guides.

**Comment by BernatBC** [comment]
Ok, I have just done it.

File: content/posts/getting-started/prepare-site/index.md [hunk 1]
```diff
@@ -310,203 +310,17 @@ badges:
 
 Put the `resume.pdf` file in `/static/files` directory of your repository. You can find the `about.yaml` file used in the example site from [here](https://github.com/hugo-toha/hugo-toha.github.io/blob/source/data/en/sections/about.yaml).
 
-###### Skills Section
+###### Add other sections
```
Comment (hossainemruz) [inline content/posts/getting-started/prepare-site/index.md:313]: Now, it is much better. Thank you @BernatBC. You can simplify the list with just links. No other details is necessary. For example:

```markdown
In order to keep this post short, we have splitted it in different posts. Here, is the list of posts that will show you how to configure the other sections:

- [Configuring About Section](/posts/configuration/sections/about/).
- [Configuring Education Section](/posts/configuration/sections/education/).
- [Configuring Experiences Section](/posts/configuration/sections/experiences/).
- [Configuring Projects Section](/posts/configuration/sections/projects/).
- [Configuring Recent Posts Section](/posts/configuration/sections/recent-posts/).
- [Configuring Achievements Section](/posts/configuration/sections/achievements/).
- [Configuring Skills Section](/posts/configuration/sections/skills/).
```

**Review by hossainemruz — APPROVED** [review]
Perfect! Thank you @BernatBC.

**Pull request merged**